assets/js/main.js

```diff
@@ -541,6 +546,27 @@ function directory(jQuery) {
 //                                          '</span>');
 //   }
   });
+
+  $.ajax({
+    async: false,
+    type: 'GET',
+    url: md5Url,
+    success: function(poster) {
+      $.ajax({
+        async: false,
+        type: 'GET',
+        url: md5OverrideUrl,
+        success: function(overrides) {
+          poster["posters"].map(
+              function (e, i) {
+                Object.assign(e, overrides["posters"][i]);
+              });
```

Comment:
Thank you @christian-monch !  Things are moving too fast,so it might need tune up now:

From my understanding of the logic here, 
- if `i` is still and index within list, it would no longer work as of f6a88eef90e720d9d7e9bc663facec10cb5941df where I RFed code to accommodate an additional input SoftwareDemos.tsv.  So I now in Python create a "proper" mapping based on the "number" field.  So something similar should be done here
- does `Object.assign(e, overrides["posters"][i])` "update" or "assigns" the override?  overrides contain only overrides, not all the fields

Reply:
1. Did not see the updated mapping, was referring to the zip-version. Will change that, should be easy.
2. How overrides work is understood. ;-) assign does an "update", i.e only fields in overrides["posters"][i] will be modified in e.

Reply:
GREAT!
Even though overrides became less important since we did populate majority of entries, I still think it would be useful, so I will keep an eye for the update!

Reply:
Has been discussed and agreed upon. Equivalent code now in force-pushed commit 030d63a